src/main/java/TicTacToe/UserInterface.java

```diff
@@ -1,11 +1,11 @@
 package TicTacToe;
 
 public class UserInterface {
-    public void startGame() {
+    public static void startGame() {
```

Comment:
Albo wszystkie metody static (wtedy ta klasa będzie FooBarUtils) albo żadna (i ta kalsa to service)